index.html

```diff
@@ -31,8 +32,272 @@ <h2 class="headline-page-h2">I'am a Software Developer</h2>
+        <p>see project</p>
+      </div>
+    </div>
+
+    <div class="content_2" id="c6">
+      <div class="card">
+        <h1>Profesonal Art <br />Printing Data</h1>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+      </div>
+      <div class="technology">
+        <ul>
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+      </div>
+      <div class="btn_yellow">
+        <p>see project</p>
+      </div>
+    </div>
+
+    <div class="content_3">
+      <div>
+        <h1>About me</h1>
+        <p>
+          Hello I'am a Software Developer! I can help you build a product,
+          feature or website Look through some of my work and experience! if you
+          like what you see and have a project you need coded, don't hestiate to
+          contacta me.
+        </p>
+      </div>
+      <div class="btn_yellow" id="yellow">
+        <p>see project</p>
+      </div>
+
+      <div class="bottom_link">
+        <h2>
+          <img src="images/Rectangle55.png" alt="" class="image55" /> Language
+        </h2>
+        <ul class="list">
+          <li>Javascript</li>
+          <li>Ruby</li>
+          <li>Html</li>
+          <li>Css</li>
+        </ul>
+
+        <div>
+          <h2>
+            <img src="images/Rectangle56.png" alt="" class="image55" />
+            Frameworks
+          </h2>
+          <ul class="list">
+            <li>Bootstrap</li>
+            <li>Ruby on Rails</li>
+            <li>RSPec</li>
+            <li>CapyBara</li>
+            <li>Selenium</li>
+          </ul>
+          <div>
+            <h2>
+              <img src="images/Rectangle57.png" alt="" class="image55" /> Skills
+            </h2>
+            <ul class="list">
+              <li>Codekit</li>
+              <li>Github</li>
+              <li>Codepen</li>
+              <li>Gitlab</li>
+              <li>Terminal</li>
+            </ul>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div class="content_4">
+      <p>
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in.
+      </p>
+      <div class="form">
+        <form id="fs-frm" name="simple-contact-form">
+          <input
+            type="text"
+            name="name"
+            id="full-name"
+            placeholder="Full name"
+            required=""
+          />
+          <input
+            type="email"
+            name="_replyto"
+            id="email-addres"
+            placeholder="Email address"
+            required=""
+          />
+          <textarea rows="5" name="message" id="message" required="">
+            Enter text here</textarea
+          >
+          <input
+            type="hidden"
+            name="_subject"
+            id="email-subject"
+            value="Contact Form Submission"
+          />
```

Comment:
- [ ] I don't know this what this does and I think it's not necessary, so please remove it.